bin/pt-kill

```diff
@@ -7373,17 +7373,42 @@ sub main {
 
          MATCHING_QUERY:
          foreach my $query ( @queries ) {
+            my $qinfo = ($query->{Info} || 'NULL' );
```

Comment:
I have moved this up outside of the `if ( $o->get('print') ) ...` scope so that it's also available in the `if ( $o->get('log-json') ) ...` scope; the downside is that it is now executed unnecessarily when neither `--print` or `--log-json` are specified.